RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreationAlgo.cc

```diff
@@ -265,7 +265,7 @@ void CaloTowersCreationAlgo::setGeometry(const CaloTowerTopology* cttopo, const
     std::vector<int> tower28depths;
     int ndepths, startdepth;
     theHcalTopology->getDepthSegmentation(theHcalTopology->lastHERing()-1,tower28depths);
-    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,ndepths,startdepth);
+    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,1,1,ndepths,startdepth);
```

Comment:
It is z= +1 and phi = 63..66. I am looking into possible changes in the CaloTowerCreationAlgo - we have to introduce zside, phi dependence there